cluster-autoscaler/core/autoscaling_context.go

```diff
@@ -140,14 +140,14 @@ func NewAutoscalingContext(options AutoscalingOptions, predicateChecker *simulat
 	logEventRecorder *utils.LogEventRecorder, listerRegistry kube_util.ListerRegistry) (*AutoscalingContext, errors.AutoscalerError) {
 
 	cloudProviderBuilder := builder.NewCloudProviderBuilder(options.CloudProviderName, options.CloudConfig, options.ClusterName, options.NodeAutoprovisioningEnabled)
-	cloudProvider := cloudProviderBuilder.Build(cloudprovider.NodeGroupDiscoveryOptions{
-		NodeGroupSpecs:             options.NodeGroups,
-		NodeGroupAutoDiscoverySpec: options.NodeGroupAutoDiscovery},
+	cloudProvider := cloudProviderBuilder.Build(
+		cloudprovider.NodeGroupDiscoveryOptions{NodeGroupSpecs: options.NodeGroups, NodeGroupAutoDiscoverySpecs: options.NodeGroupAutoDiscovery},
 		cloudprovider.NewResourceLimiter(
 			map[string]int64{cloudprovider.ResourceNameCores: int64(options.MinCoresTotal), cloudprovider.ResourceNameMemory: options.MinMemoryTotal},
-			map[string]int64{cloudprovider.ResourceNameCores: options.MaxCoresTotal, cloudprovider.ResourceNameMemory: options.MaxMemoryTotal}))
-	expanderStrategy, err := factory.ExpanderStrategyFromString(options.ExpanderName,
-		cloudProvider, listerRegistry.AllNodeLister())
+			map[string]int64{cloudprovider.ResourceNameCores: options.MaxCoresTotal, cloudprovider.ResourceNameMemory: options.MaxMemoryTotal},
+		),
+	)
+	expanderStrategy, err := factory.ExpanderStrategyFromString(options.ExpanderName, cloudProvider, listerRegistry.AllNodeLister())
```

Comment:
Why change this formatting?

Reply:
Unintentional, I think. Pretty sure I had split it out a bit so I could better follow what it was doing and forgot to put it back. Happy to revert.